File: src/Workflow.Web.Service/Workflow/app/model/common/worklists/Activity.js
Ext.define('Workflow.model.common.worklists.Activity', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WorkflowId', type: 'string', mapping: 'WorkflowId' },
        { name: 'Type', type: 'string', mapping: 'Type' },
        { name: 'Prefix', type: 'string', mapping: 'Prefix' },
        { name: 'Name', type: 'string', mapping: 'Name' },
        { name: 'DisplayName', type: 'string', mapping: 'DisplayName' },
        { name: 'ActCode', type: 'string', mapping: 'ActCode' },
        { name: 'IconIndex', mapping: 'IconIndex' },
        { name: 'Sequence', mapping: 'Sequence' },
        { name: 'Active', mapping: 'Active' }
    ],
    proxy:{
        type:'rest',
        url: Workflow.global.Config.baseUrl + 'api/worklists/activities',
        reader:{
                type: 'json',
                rootProperty: 'Records'
        }
    }
});

File: src/Workflow.Domain/Entities/Core/Activity.cs
namespace Workflow.Domain.Entities.Core
{
    public partial class Activity
    {
        public int Id { get; set; }

        public int? WorkflowId { get; set; }

        public string Type { get; set; }
        
        public string Name { get; set; }

        public string DisplayName { get; set; }

        public string ActCode { get; set; }
        
        public string Property { get; set; }
        
        public int? Sequence { get; set; }

        public bool? Active { get; set; }

    }
}
